packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -13,10 +13,19 @@ import path from "path";
 
 let babelHelpers = eval(buildExternalHelpers(null, "var"));
 
-function wrapPackagesArray(type, names) {
+function wrapPackagesArray(type, names, optionsDir) {
   return (names || []).map(function (val) {
     if (typeof val === "string") val = [val];
-    val[0] = __dirname + "/../../babel-" + type + "-" + val[0];
+
+    // relative path (outside of monorepo)
+    if (val[0][0] === ".") {
+      val[0] = path.resolve(optionsDir, val[0]);
```

Comment:
Testing in babel-preset-env https://github.com/babel/babel-preset-env/pull/28 with `"../../../../lib"` under `test/fixtures/preset-options/modules-false/options.json`

optionsDir: `/Users/hzhu/dev/babel-preset-env/test/fixtures/preset-options/modules-false`
val[0]: `../../../../lib`
path.resolve: `/Users/hzhu/dev/babel-preset-env/lib`